dev/SwipeControl/SwipeControl.cpp

```diff
@@ -721,15 +721,21 @@ void SwipeControl::DetachDismissingHandlers()
         {
             auto xamlRootContent = xamlRoot.Content();
 
-            if (m_onXamlRootPointerPressedEventHandler)
+            if (auto&& handler = m_onXamlRootPointerPressedEventHandler.get())
             {
-                xamlRootContent.RemoveHandler(winrt::UIElement::PointerPressedEvent(), m_onXamlRootPointerPressedEventHandler.get());
+                if (xamlRootContent)
+                {
+                    xamlRootContent.RemoveHandler(winrt::UIElement::PointerPressedEvent(), handler);
```

Comment:
I believe there's now a helper that lets you use the AddHandler/RemoveHandler as a revoker pattern instead. Can we switch this code to that instead?

Reply:
It's in RoutedEventHelpers.h and it looks liek PointerPressed isn't templated in there yet (but KeyDown is, so you should be able to copy that relatively easily).

Reply:
sweet